### DISCUSSION TITLE
Found this blog, maybe it helps someone

### DISCUSSION THREAD

**Discussion by snailcatcher** [OpenNeptune3D/OpenNept4une#84] [Show and tell]
https://r6e.github.io/neptune_4/hardware.html

**Comment by steve8x8** [comment]
The underlying git repository is https://github.com/r6e/r6e.github.io.git, and `git log -p _neptune4` tells me the date of the last modification: Sep 15, 2023. Much has changed since...

**Comment by halfmanbear** [comment]
Got some interesting info but yes much more is understood now it got some things very close though
